projects/packages/jetpack-mu-wpcom/src/features/wpcom-themes/css/ui-fixes.css

```diff
@@ -0,0 +1,3 @@
+.theme-browser .button:visited {
+	color: #fff;
```

Comment:
It seems that the original issue is caused by a generic style set by the help center CSS. Can we maybe fix that in the help center?

<img width="1277" alt="Screenshot 2024-05-13 at 10 42 05" src="https://github.com/Automattic/jetpack/assets/1233880/b6793ac2-7995-468f-9ace-ecefb3f7192a">

Reply:
I tried, but it doesn't seem as trivial. The style isn't actually part of the help center code.